projects/storefrontapp-e2e-cypress/cypress.json

```diff
@@ -6,6 +6,11 @@
   "env": {
     "CLIENT_ID": "mobile_android",
     "CLIENT_SECRET": "secret",
-    "API_URL": "https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002"
+    "API_URL": "https://dev-com-17.accdemo.b2c.ydev.hybris.com:9002",
+    "OCC_PREFIX": "/rest/v2",
+    "BASE_SITE": "/electronics-spa",
```

Comment:
IMO the base site is not specific to the environment, but to the sample data, I'm not convinced it's a good place to store this const. I'd rather have it statically in some TS enum.

Base site is also hardcoded in lines 12 and 14

Reply:
I agree that the base site is not specific to the environment, however, the reason base_site needs to be included here is because in the future we will have multiple e2e that covers other base_site. Our primary base site is electronics-spa, therefore it is our default. 

When working with other base_sites for other e2e that will re-use other helper functions from other class, we will have a way to facilitate the transition. As explained in our messages, if you look at the modified file for `apparel-checkout-flow` or `apparel-checkout-as-guest`, you can see how easy it is to modify it without affecting the e2e code base. Also, to recall, when modifying the `environment variables`, it will not persist to other tests as it is contained within each session that cypress starts individually for each test.

My main argument for going with this flow is, it will modify a lot of functions and won't be re-usable from the getgo. If you want to re-use a function that needs the other `basesite`, you would need to pass in a parameter to that function that will allow the `basesite` to work, for example `waitForPage` function and a lot others. IF we were to create a file, and create an enum of basesites, we would have to go through that route, and modify individually each function to allow re-usability in the e2e test as I have experienced when adding the apparel-e2e to our suite.